src/message.rs

```diff
@@ -145,6 +148,25 @@ macro_rules! message_args_impl {
                 })
             }
 
+            #[cfg(all(feature="gnustep_runtime",
+                      not(any(target_arch = "arm",
+                              target_arch = "x86",
+                              target_arch = "x86_64"))))]
+            unsafe fn send<T, R>(self, obj: *mut T, sel: Sel) -> R
+                   where T: Message, R: Any {
+                    let mut receiver = obj as *mut Object;
+                    let nil: *mut Object = ::std::ptr::null_mut();
+                    let ref slot = *runtime::objc_msg_lookup_sender(&mut receiver as *mut *mut Object, sel, nil);
```

Comment:
Why can't this use `objc_msgSend`? I've been trying to read up on gnustep's concept of slots but haven't found a great introduction in the documentation. Does gnustep not support `objc_msgSend`?

Otherwise the only reason to use the slot lookup seems to be so that you can keep the slot and call the method repeatedly, but since we get the slot anew each time that doesn't seem like it'd be a benefit.